### ISSUE TITLE
Version 0.17.4 does not run on Windows 11 Pro

### ISSUE THREAD

**Issue by Gunnl** [thinkle/gourmet#1038]
### Steps to Reproduce
1. Download version 0.17.4 .msi
2. install
3. Attempt to run
4.

### Expected Behavior
The tool should open, and run without problems

### Current Behavior
and exception window is shown and the tool closes.
![image](https://user-images.githubusercontent.com/352718/208263679-6cf21ea0-f3a2-4179-9a35-507441c725c7.png)

### Environment

* Operating System: windows 11 Pro
* Version or commit id: 0.17.4
* Installed as Flatpak or Python package?: .MSI package

**Comment by 1330918** [comment]
Will this ever be fixed? I can currently only use Gourmet on my old laptop, but I don't know for how long that one will still work. I have tons of recipes and export them to pdf. I really like the interface of Gourmet recipes and would love to be able to use it in the future with windows 11. @thinkle @pounamu

**Comment by 1330918** [comment]
@thinkle

**Comment by FriedrichFroebel** [comment]
Gourmet itself will most likely not receive any further updates itself AFAIK and is focusing on a new web-based platform, although I do not know the current status. The Gourmand fork (https://github.com/GourmandRecipeManager/gourmand) has been stale for some time as well, although I recently started trying to revive it. Nevertheless, getting it to run on Windows might be a bit more complicated as there is no installer/standalone executable available for now.

**Comment by allenerenee** [comment]
Gourmet has actually started freezing if I leave it open too long in Linux.
I'm currently porting everything out and into Mealie - but I have to
copy-paste, because one of the things that's been broken a really long time
now is the export functionality. Get your stuff out while you can....

**Comment by thinkle** [comment]
Yes I'm happy to help you try to port an old database to the web version if
you want. It's live at www.gourmetrecipemanager.com though mainly just for
my personal recipes at this point 😂 that said, it should in theory work.
Login is a little finicky and handled through a netlify interface but once
you're set up it's there. I forget the details but I believe the desktop
gourmet has an xml export format which I could in theory export. Iirc I
exported my own recipes to a json format which was then easy to import into
the new system.

Tom

**Comment by 1330918** [comment]
Thanks for the quick replies! I will have a look at the web-based version. Not a huge fan of those, since databases can be large GHG emitters and I am a bit hesitant about that.
